rshim.spec.in

```diff
@@ -96,6 +96,9 @@ fi
 %{_mandir}/man8/bfb-install.8.gz
 
 %changelog
+* Fri, 12 Apr 2024 Penghe Geng <pgeng@nvidia.com> - 2.0.27
+- bfb-install: Fix incorrect IP address resolution for multi-hop routing
```

Comment:
rhel/rshim.spec.in should be updated as well

Reply:
Thanks. Added.